apps/src/sites/studio/pages/projects/index.js

```diff
@@ -48,10 +48,12 @@ $(document).ready(() => {
           projectCount={projectsData.projectCount}
           specialAnnouncement={specialAnnouncement}
         />
-        <ProjectsGallery
-          limitedGallery={projectsData.limitedGallery}
-          canShare={projectsData.canShare}
-        />
+        <div className={'main container'}>
```

Comment:
Hopefully ordering doesn't matter, but could do `container main` to match the [existing order in HAML](https://codedotorg.slack.com/archives/C1B3PNDL7/p1595278385372100).  We also appear to make use of [Classnames](https://github.com/JedWatson/classnames) which might be a bit more readable?

Reply:
I swapped the names. Good catch.
I didn't use `Classnames` because it seems overkill and less readable than just the raw string. But I might not understand how to use it in an elegant way, so feel free to show me how to use it.